pages/movies/[id].js

```diff
@@ -35,13 +67,17 @@ export default function MovieDetailsPage() {
           width={150}
           height={220}
         />
-        <p>Title:{movie.title}</p>
-        <p>Release Date: {movie.release_date}</p>
-        <p>Duration: {movie.runtime} min</p>
-        <p>Genre: {movie.genres[0].name}</p>
-        <p>Rating: {movie.vote_average}</p>
+        <ul>
+          <Title>{movie.title}</Title>
+          <List>{movie.release_date}</List>
+          <List>{movie.runtime} min</List>
+          <List>{movie.genres[0].name}</List>
+          <List>{movie.vote_average}</List>
+        </ul>
         <p>{movie.overview}</p>
```

Comment:
why do you have a component called MovieDetails and this code in the MovieDetailsPage? This looks like duplicated code to me.